packages/core/src/hydration/utils.ts

```diff
@@ -344,6 +341,14 @@ export function markRNodeAsSkippedByHydration(node: RNode) {
   ngDevMode.componentsSkippedHydration++;
 }
 
+export function countBlocksSkippedByHydration(injector: Injector) {
+  const transferState = injector.get(TransferState);
+  const nghDeferData = transferState.get(NGH_DEFER_BLOCKS_KEY, {});
+  if (ngDevMode) {
+    ngDevMode.deferBlocksWithIncrementalHydration += Object.keys(nghDeferData).length;
```

Comment:
nit: we can probably just re-assign the value:
```suggestion
    ngDevMode.deferBlocksWithIncrementalHydration = Object.keys(nghDeferData).length;
```